prettier.config.js

```diff
@@ -3,4 +3,5 @@
 module.exports = {
   semi: false,
   singleQuote: true,
+  trailingComma: 'es5',
```

Comment:
The default is `'es5'`: https://prettier.io/docs/en/options.html#trailing-commas

I would revert this unnecessary addition (but it doesn't do anything, so it's also OK to me to keep it).